javadoc/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'java'
+}
+
+java {
```

Comment:
can we make _allllll_ of this file predicated on the property?  will need to change to `apply plugin: 'java'`, but otherwise should make it inert for anyone running `./gradlew jar`. ....tho, I suppose we _do_ universally apply plugins from root project.

I suspect you may need to either a) use a system prop instead of gradle prop (so settings.gradle can see it), or b) exculde this subproject from default getting java plugin applied, and then make everything inside it conditional (leave project in source root).

Whatever we do, we should leave breadcrumbs for any poor person who wants to edit this beast to get IDE support.

Reply:
I think it is already predicated on the property in settings.

That said, I don't know how to mix-n-match IDE support w/ IJ. The java 11 code shows up as red errors for me in IJ, and the idea block setting the language level to 11 doesn't seem to change anything.